core/DataTable/Row.php

```diff
@@ -521,12 +521,18 @@ private function getColumnValuesMerged($operation, $thisColumnValue, $columnToSu
                 $newValue = null;
                 break;
             case 'max':
-                $newValue = max($thisColumnValue, $columnToSumValue);
+                if (null === $thisColumnValue || false === $thisColumnValue || '' === $thisColumnValue) {
```

Comment:
This might be a bit cryptic
```suggestion
                if (0 == $thisColumnValue && 0 !== $thisColumnValue) {
```
but this might be easier to read
```suggestion
                if (in_array($thisColumnValue, [null, false, ''], true)) {
```